src/main/scala/org/ua/ilm/datamining/crawlers/youtube/parse/ChannelParser.scala

```diff
@@ -1,18 +1,23 @@
-package org.ua.ilm.Entity
+package org.ua.ilm.datamining.crawlers.youtube.parse
 
 import com.fasterxml.jackson.databind.ObjectMapper
 import com.fasterxml.jackson.module.scala.experimental.ScalaObjectMapper
+import org.ua.ilm.Entity.Channel
 
-object YoutubeChannelParser {
+class ChannelParser {
   val mapper = new ObjectMapper() with ScalaObjectMapper
 
-  def parse(string: String): Channel = {
-    val jsonNode = mapper.readTree(string)
+  def parse(jsonChannel: String): Channel = {
+    val jsonNode = mapper.readTree(jsonChannel)
     val title = jsonNode.findValue("title").asText()
     val keywords = jsonNode.findValue("keywords").asText()
     val subscriberCount = jsonNode.findValue("subscriberCount").asLong()
     val viewCount = jsonNode.findValue("viewCount").asLong()
     val description = jsonNode.findValue("description").asText()
     Channel(title, keywords, subscriberCount, viewCount, description)
   }
+  def getChannels(channelsOfJson: Array[String]):Array[Channel] ={
+    val listChannels = for{item <- channelsOfJson}yield parse(item)
+    listChannels
+  }
```

Comment:
ToDo: change in callable solution according to review comments. Than you should rebase this branch on callable branch to actualize this code.